@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -701,26 +694,6 @@
                             2
                         ]
                     },
-                    "secondaryWidth": {
```

Comment:
Why removing this? The actually help to reuse the same vertex buffers for 2 different objects.

Reply:
Yep, this works on solid-line, but not on dashed-line.

Reply:
You're right, didn't realize.